src/protagonist/API.Tests/Settings/ApiSettingsTests.cs

```diff
@@ -51,4 +51,94 @@ public void ApiSettings_LegacySupportEnabledForSingleCustomer_WhenDefaultLegacyD
         settings.GetCustomerSettings(2).LegacySupport.Should().BeTrue();
         settings.GetCustomerSettings(2).NovelSpaces.Should().Contain("1");
     }
+    
+    [Fact]
+    public void LegacyModeEnabled_LegacySupportEnabledForAllCustomers_byDefault()
+    {
+        // Arrange
+        var settings = new ApiSettings();
+        
+        // Assert
+        settings.LegacyModeEnabled(1,1).Should().BeTrue();
+    }
+    
+    [Fact]
+    public void LegacyModeEnabled_LegacySupportDisabledForAllCustomers_WhenDefaultLegacyDisabled()
+    {
+        // Arrange
+        var settings = new ApiSettings
+        {
+            DefaultLegacySupport = false
+        };
+        
+        // Assert
+        settings.LegacyModeEnabled(1,1).Should().BeFalse();
+    }
+    
+    [Fact]
+    public void LegacyModeEnabled_LegacySupportEnabledForSingleCustomer_WhenDefaultLegacyDisabled()
+    {
+        // Arrange
+        var settings = new ApiSettings
+        {
+            DefaultLegacySupport = false
+        };
+
+        settings.CustomerOverrides.Add("2", new CustomerOverrideSettings()
+        {
+            LegacySupport = true,
+            NovelSpaces = new List<string>()
+            {
+                "1"
+            }
+        });
+        
+        // Assert
+        settings.LegacyModeEnabled(1, 1).Should().BeFalse();
+        settings.LegacyModeEnabled(2, 1).Should().BeFalse();
+        settings.LegacyModeEnabled(2, 2).Should().BeTrue();
+    }
+    
+    [Fact]
+    public void LegacyModeEnabled_LegacySupportDisabledForSpace_WhenDefaultLegacyEnabled()
+    {
+        // Arrange
+        var settings = new ApiSettings();
+        
+        settings.CustomerOverrides.Add("2", new CustomerOverrideSettings()
+        {
+            LegacySupport = true,
+            NovelSpaces = new List<string>()
+            {
+                "1"
+            }
+        });
+        
+        settings.CustomerOverrides.Add("3", new CustomerOverrideSettings()
+        {
+            LegacySupport = false
+        });
+        
+        // Assert
+        settings.LegacyModeEnabled(1, 1).Should().BeTrue();
+        settings.LegacyModeEnabled(2, 1).Should().BeFalse();
+        settings.LegacyModeEnabled(2, 2).Should().BeTrue();
+    }
+
+    [Fact]
+    public void LegacyModeEnabled_LegacySupportDisabledForCustomer_WhenDefaultLegacyEnabled()
+    {
+        // Arrange
+        var settings = new ApiSettings();

```

Comment:
Reading this test - is the default set in code for `LegacySupport = true`? If so I think it should be `false`. 

A newly spun up DLCS without any prior customers shouldn't support legacy formats. We will want to set it to `true` as a default in some instances, but that should be via config for existing installs with existing customers. Apologies, I must have missed that on previous PR!

Reply:
yeah, must have misinterpreted! - will get it changed so it's false by default